plugins/base/src/main/kotlin/translators/documentables/DefaultDocumentableToPageTranslator.kt

```diff
@@ -8,10 +9,11 @@ import org.jetbrains.dokka.transformers.documentation.DocumentableToPageTranslat
 import org.jetbrains.dokka.utilities.DokkaLogger
 
 class DefaultDocumentableToPageTranslator(
+    private val configuration: DokkaBaseConfiguration?,
     private val commentsToContentConverter: CommentsToContentConverter,
     private val signatureProvider: SignatureProvider,
     private val logger: DokkaLogger
```

Comment:
Just wondering if we should pass the `DokkaContext` here instead, as it would allow us to reduce the number of parameters, I don't have a strong opinion though

Reply:
Yep. Every one of the parameters can be extracted from the context in just one line. I also think most of the not trivial extensions are using `[...] providing ::XYZExtension` pattern, so passing only context here would be more consistent.